includes/ucf-resource-search-common.php

```diff
@@ -204,55 +204,54 @@ public static function ucf_resource_search_display( $params, $search_data ) {
 								</h2>
 							</label>
-							<input type="text" id="resource-search-input" name="resource-search-input" class="form-control form-control-lg resource-search-input" placeholder="<?php echo $params['default_search_text']; ?>">
+							<input type="text" id="resource-search-input" name="resource-search-input" class="form-control resource-search-input" placeholder="<?php echo $params['default_search_text']; ?>">
 						</div>
 					</form>
 				</div>
 				<div class="resource-search-results"></div>
 				<?php if ( $params['show_sorting'] ) { ?>
 				<div class="btn-group resource-search-sorting">
 					<button class="btn btn-default<?php if ( $params['default_sorting'] == 'term' ) echo ' active'; ?>">
-						<span class="glyphicon glyphicon-list-alt"></span>
 					</button>
 					<button class="btn btn-default<?php if ( $params['default_sorting'] == 'alpha' ) echo ' active'; ?>">
-						<span class="glyphicon glyphicon-font"></span>
 					</button>
 				</div>
 				<?php } ?>
 
-				<div class="btn-toolbar jump-to-list mt-5 mb-5" role="toolbar" aria-label="Jump To List">
+				<div class="btn-toolbar jump-to-list" role="toolbar" aria-label="Jump To List">
 					<div class="btn-group" role="group">
-						<a href="#jump-to-a" class="btn btn-default p-1">A</a>
-						<a href="#jump-to-b" class="btn btn-default p-1">B</a>
-						<a href="#jump-to-c" class="btn btn-default p-1">C</a>
-						<a href="#jump-to-d" class="btn btn-default p-1">D</a>
-						<a href="#jump-to-e" class="btn btn-default p-1">E</a>
-						<a href="#jump-to-f" class="btn btn-default p-1">F</a>
-						<a href="#jump-to-g" class="btn btn-default p-1">G</a>
-						<a href="#jump-to-h" class="btn btn-default p-1">H</a>
-						<a href="#jump-to-i" class="btn btn-default p-1">I</a>
-						<a href="#jump-to-j" class="btn btn-default p-1">J</a>
-						<a href="#jump-to-k" class="btn btn-default p-1">K</a>
-						<a href="#jump-to-l" class="btn btn-default p-1">L</a>
-						<a href="#jump-to-m" class="btn btn-default p-1">M</a>
-						<br class="hidden-md-up">
-						<a href="#jump-to-n" class="btn btn-default p-1">N</a>
-						<a href="#jump-to-o" class="btn btn-default p-1">O</a>
-						<a href="#jump-to-p" class="btn btn-default p-1">P</a>
-						<a href="#jump-to-q" class="btn btn-default p-1 disabled">Q</a>
-						<a href="#jump-to-r" class="btn btn-default p-1">R</a>
-						<a href="#jump-to-s" class="btn btn-default p-1">S</a>
-						<a href="#jump-to-t" class="btn btn-default p-1">T</a>
-						<a href="#jump-to-u" class="btn btn-default p-1">U</a>
-						<a href="#jump-to-v" class="btn btn-default p-1">V</a>
-						<a href="#jump-to-w" class="btn btn-default p-1">W</a>
-						<a href="#jump-to-x" class="btn btn-default p-1 disabled">X</a>
-						<a href="#jump-to-y" class="btn btn-default p-1 disabled">Y</a>
-						<a href="#jump-to-z" class="btn btn-default p-1 disabled">Z</a>
+						<a href="#jump-to-a" class="btn btn-default">A</a>
```

Comment:
One thing that you might do to make the code a little more minimalist would be to loop through a range instead of listing out every letter:

```php
<?php foreach alpha in range( 'a', 'm' ) : ?>
    <a href="#jump-to-<?php echo alpha; ?>" class="btn btn-default"><?php echo strtoupper( alpha ); ?></a>
<?php endfor; ?>
```

Rinse and repeat for N-Z. It would at least give the advantage of being much easier to update if we change a style on all of the buttons at least.